controllers/topics/index.js

```diff
@@ -17,7 +17,14 @@ const updateTopic = require('./updateTopic');
 const create = require('./create');
 const getFollowedTopic = require('./getFollowedTopic');
 const getLatestPost = require('./getLatestPost');
-const {Topics, UserTopic, UserTopicHistory, sequelize, User} = require('../../databases/models');
+const {
+  Topics,
+  UserTopic,
+  UserTopicHistory,
+  sequelize,
+  User,
+  TopicInvitations
+} = require('../../databases/models');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been expanded to include `TopicInvitations` from the models. This is necessary for the new feature.
```diff
- const {Topics, UserTopic, UserTopicHistory, sequelize, User} = require('../../databases/models');
+ const {
+   Topics,
+   UserTopic,
+   UserTopicHistory,
+   sequelize,
+   User,
+   TopicInvitations
+ } = require('../../databases/models');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->